### PR TITLE
Sentry integration

### PR DESCRIPTION
**Pull request by christiandrey**
Add Sentry integration to browser extension

### CODE REVIEW

File: src/manifest.chrome.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Toggl Plan: Project Planning Calendar",
-  "version": "2.0.3",
+  "version": "2.0.4",
```
Comment (kelonye) [inline src/manifest.chrome.json:3]: I'd bump this in a separate commit on master / release PR.

File: src/manifest.firefox.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Toggl Plan: Project Planning Calendar",
-  "version": "2.0.2",
+  "version": "2.0.3",
```
Comment (kelonye) [inline src/manifest.firefox.json:3]: Same ☝️

File: src/manifest.chrome.json (hunk 1 above)
Comment (christiandrey) [reply src/manifest.chrome.json:3]: Noted.

File: src/manifest.firefox.json (hunk 2 above)
Comment (christiandrey) [reply src/manifest.firefox.json:3]: Noted.

**Pull request merged**